### PR TITLE
Rfced 12 pranswer

### LINKED ISSUE #882: 12) [rfced] Section 4.1.8.1:  We had trouble with this sentence.
12) [rfced] Section 4.1.8.1:  We had trouble with this sentence.
If neither suggestion below is correct, please clarify.

Original:
 While
 this could also be done with a "sendonly" pranswer, followed by a
 "sendrecv" answer, the initial pranswer leaves the offer-answer
 exchange open, which means that the caller cannot send an updated
 offer during this time.

Suggestion #1:
 While
 this could also be done with a "sendonly" pranswer, if followed by a
 "sendrecv" answer the initial pranswer leaves the offer/answer
 exchange open, which means that the caller cannot send an updated
 offer during this time.

Suggestion #2:
 While
 this could also be done with a "sendonly" pranswer followed by a
 "sendrecv" answer, the initial pranswer leaves the offer/answer
 exchange open, which means that the caller cannot send an updated
 offer during this time.

### PR DESCRIPTION
**Pull request by ajeanmahoney**
used suggestion 2, deleted comments, fixes #882, issue 12

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -1379,37 +1360,11 @@ Suggested:
...
-
-Suggestion #2:
- While
- this could also be done with a "sendonly" pranswer followed by a
- "sendrecv" answer, the initial pranswer leaves the offer/answer
- exchange open, which means that the caller cannot send an updated
- offer during this time. -->
-
-</t>
+	    bidirectional media flow.  While this could also be done
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:1363]: Indentation seems to have changed. Is this intentional?

Comment (ajeanmahoney) [reply draft-ietf-rtcweb-jsep.xml:1363]: Are indentation changes an issue?  I just cut and paste suggestion 2 from the comments into the text.  Indents didn't quite align.

Comment (juberti) [reply draft-ietf-rtcweb-jsep.xml:1363]: For ease of reading/reviewing, I'd prefer to maintain the original indent. I can handle this with a post-PR commit if needed.

Comment (ajeanmahoney) [reply draft-ietf-rtcweb-jsep.xml:1363]: Please fix, thanks!

**Pull request merged**